changelog/README.md

```diff
@@ -2,8 +2,9 @@
 
 <!-- Note: contrary to 3.x, insert new entries *first* in their section -->
 
-### 4.0.2 (in progress)
+### 4.1.0 (in progress)
 
+- [bug] JAVA-2264: HashedWheelTimer tick duration should be >= 100 ms on Windows
```

Comment:
Just a suggestion, but would make it more clear that we're changing the default overall here, something like:

```suggestion
- [bug] JAVA-2264: Adjust HashedWheelTimer tick duration from 1 to 100 ms
```